modules/null/agent-gw-cluster-setup/setup.tftpl

```diff
@@ -0,0 +1,132 @@
+#!/bin/bash
+#set -x
+
```

Comment:
I usually prefer to put "set -e" to find "unexpected failures" in these scripts.
set -x is also important to have details of failing environments that glitch here and there

Reply:
I forgot to uncomment it, I will uncomment it and add set -e.